src/ehrdata/dt/datasets.py

```diff
@@ -524,6 +519,131 @@ def physionet2012(
+
+    This dataset was created for the purpose of a case study in the book: `Secondary Analysis of Electronic Health Records <https://link.springer.com/book/10.1007/978-3-319-43742-2>`_.
+    In particular, the dataset was used to investigate the effectiveness of indwelling arterial catheters in hemodynamically stable patients with respiratory failure for mortality outcomes.
+    The dataset is derived from MIMIC-II, the publicly-accessible critical care database. It contains summary clinical data and  outcomes for 1,776 patients.
+
+    More details on the original dataset: https://physionet.org/content/mimic2-iaccd/1.0/.
+
+
+    Examples:
+        >>> import ehrdata as ed
+        >>> edata = ed.dt.mimic_2_preprocessed()
+    """
+    _download(
+        url="https://figshare.com/ndownloader/files/39727936",
+        output_path=DEFAULT_DATA_PATH,
+        output_filename="mimic_2_preprocessed.h5ad",
+        raw_format="h5ad",
+    )
+    edata = read_h5ad(
+        filename=f"{DEFAULT_DATA_PATH}/mimic_2_preprocessed.h5ad",
+    )
+
+    return edata
+
+
+def diabetes_130_raw(
+    columns_obs_only: Iterable[str] | None = None,
+) -> EHRData:
+    """Loads the raw diabetes-130 dataset.
+
+    More details: http://archive.ics.uci.edu/ml/datasets/Diabetes+130-US+hospitals+for+years+1999-2008 [1]
+
+    Args:
+        columns_obs_only: Columns to include in `obs` only and not `X`.
+
+    Examples:
+        >>> import ehrdata as ed
+        >>> edata = ed.dt.diabetes_130_raw()
+
+    References:
+        [1] Beata Strack, Jonathan P. DeShazo, Chris Gennings, Juan L. Olmo, Sebastian Ventura, Krzysztof J. Cios, and John N. Clore, “Impact of HbA1c Measurement on Hospital Readmission Rates: Analysis of 70,000 Clinical Database Patient Records,” BioMed Research International, vol. 2014, Article ID 781670, 11 pages, 2014.
+    """
+    _download(
+        url="https://figshare.com/ndownloader/files/45110029",
+        output_path=DEFAULT_DATA_PATH,
+        output_filename="diabetes_130_raw.csv",
+        raw_format="csv",
+    )
+    adata = read_csv(
+        filename=f"{DEFAULT_DATA_PATH}/diabetes_130_raw.csv",
+        columns_obs_only=columns_obs_only,
+    )
+
+    return adata
+
+
+def diabetes_130_fairlearn(
+    columns_obs_only: Iterable[str] | None = None,
+) -> EHRData:
+    """Loads the preprocessed diabetes-130 dataset by fairlearn.
+
+    This loads the dataset from the `fairlearn.datasets.fetch_diabetes_hospital` function.
+
+    More details: http://archive.ics.uci.edu/ml/datasets/Diabetes+130-US+hospitals+for+years+1999-2008 [1]
+
+    Preprocessing: https://fairlearn.org/v0.10/api_reference/generated/fairlearn.datasets.fetch_diabetes_hospital.html#fairlearn.datasets.fetch_diabetes_hospital [2]
+
+    Args:
+        columns_obs_only: Columns to include in `obs` only and not `X`.
+
+
+    Examples:
+        >>> import ehrdata as ed
+        >>> edata = ed.dt.diabetes_130_fairlearn()
+
+    References:
```

Comment:
All of these should be cited properly. We have a references.bib. Put stuff there and cite it